src/vos/vos_io.c

```diff
@@ -1020,7 +1029,7 @@ vos_reserve_single(struct vos_io_context *ioc, uint16_t media,
 
 	D_ASSERT(ioc->ic_umoffs_cnt > 0);
 	umoff = ioc->ic_umoffs[ioc->ic_umoffs_cnt - 1];
-	irec = (struct vos_irec_df *) umem_off2ptr(vos_obj2umm(obj), umoff);
+	irec = (struct vos_irec_df *) umem_off2ptr(vos_cont2umm(cont), umoff);
```

Comment:
This is probably my own ignorance, but how can the offset off of an object umem_base be the same as the offset off a container umem_base?

Reply:
The first argument is the umem_instance.  It’s just the struct that points at the pool.  It’s needed to interpret the offset.  The pool associated with the container is the same pool as the one associated with the object.   In fact, if you look at the definition of the object version, it’s something like obj->cont->umm